src/components/Alerts/src/stories/Alerts.stories.mdx

```diff
@@ -0,0 +1,159 @@
+import { ArgsTable, Canvas, Description, Meta, Story } from "@storybook/addon-docs/blocks";
```

Comment:
If the upgrade storybook PR merges before yours, change this import to 
`from "@storybook/addon-docs"`